types/src/account_state.rs

```diff
@@ -81,6 +81,8 @@ impl AccountState {
             .filter_map(|(k, v)| match Path::try_from(k) {
                 Ok(Path::Resource(struct_tag)) => Some((struct_tag, v.as_ref())),
                 Ok(Path::Code(_)) | Err(_) => None,
+                // TODO: consider flattening into resources, but this isn't currently used
```

Comment:
Is this TODO still needed now that the list resources API has been updated to flatten resource groups?

Reply:
yes, this reads directly from storage... but we should probably delete something that isn't used :P